Tools/LyTestTools/ly_test_tools/log/log_monitor.py

```diff
@@ -81,6 +81,13 @@ def monitor_log_for_lines(self,
         :param timeout: int time in seconds to search for expected/unexpected lines before raising LogMonitorException.
         :return: True if monitoring succeeded, raises a LogMonitorException otherwise.
         """
+        waiter.wait_for(
```

Comment:
It might be better if this functionality were optional rather than enforced on all code using it.
Reasoning: in the past I believe this function or the class `__init__` had functionality like this, but it was removed. Making it a new parameter with a default value should make it opt in and not break any existing calls that use `monitor_log_for_lines()` since it will have a default value set (ideally a boolean `False` that can be toggled `True` if a user desires)

Reply:
actually the time is sufficiently low for it to not matter (5 seconds)